plugins/woocommerce/phpcs.xml

```diff
@@ -50,6 +50,16 @@
 		<exclude-pattern>tests/</exclude-pattern>
 	</rule>
 
+	<rule ref="Suin.Classes.PSR4">
```

Comment:
TBD: Generic.PHP.RequireStrictTypes rule over bash-linting (see p1721196946974449-slack-C060B25RT41 for evaluating the change)